frontend/src/components/Header/Header.jsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react'
+import FoxticketLogo from '../../assets/images/logos/Foxticket.png'
+import Logout from './Logout';
+import { Link } from "react-router-dom";
+import { connect } from 'react-redux';
+import './Header.styles.sass'
+
+function Header(props) {
+  return (
+    <nav className="NavbarItems" >
+      <img src={FoxticketLogo} alt='navigation bar logo'></img>
+      <h3 className="nav-text">Foxticket</h3>
+      <ul className="nav-menu">
+      { 
+        props.type ==='admin' &&
```

Comment:
@tamas-w you don't have a `type` property in the props, but it'd be better to have an `isAdmin` boolean property coming from redux